### PR TITLE
Scrum 11

### PR DESCRIPTION
**Pull request by Hsbalazs**
Created frontend CI

### CODE REVIEW

**Review by carlos-ponton — APPROVED** [review]
Approved after verifying the changes with Balázs

**Review by AnnaUgrai — APPROVED** [review]

**Review by AnnaUgrai — CHANGES_REQUESTED** [review]
Why did you changed the properties for the frontend CI ?

**Comment by Hsbalazs** [comment]
Application properties was changed because some lines were duplicated at my file and I pushed it. Now it is fine.

File: backend/src/main/resources/application.properties [hunk 1]
```diff
@@ -1,11 +1,11 @@
 spring.application.name=backend
 
-spring.datasource.url=jdbc:postgresql://localhost:5432/db
+spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
```
Comment (markkovari) [inline backend/src/main/resources/application.properties:3]: This PRs scope is the frontend and its CI. Why do we need this change?
And on the other hand why the change? And how come it is not configurable.

File: .github/workflows/frontend-ci.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      run:
+        working-directory: frontend
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '22'
+      - name: Install dependencies
+        run: npm install
```
Comment (markkovari) [inline .github/workflows/frontend-ci.yaml:29]:
```suggestion
        run: npm ci
```

**Review by markkovari — CHANGES_REQUESTED** [review]
image

Looking strong. Only a few questions from my side.

File: .github/workflows/frontend-ci.yaml (hunk 2 above)
Comment (markkovari) [reply .github/workflows/frontend-ci.yaml:29]: We use this most of the times. https://stackoverflow.com/questions/52499617/what-is-the-difference-between-npm-install-and-npm-ci

File: backend/src/main/resources/application.properties (hunk 1 above)
Comment (markkovari) [reply backend/src/main/resources/application.properties:3]: Well, no worries we can fix it later.

Comment (markkovari) [reply backend/src/main/resources/application.properties:3]: Oh yeah, it ended up being a conflict, so all you have to do is just to resolve that.

File: .github/workflows/frontend-ci.yaml (hunk 2 above)
Comment (carlos-ponton) [reply .github/workflows/frontend-ci.yaml:29]: @Hsbalazs Can you complete this change requested by Mark?

**Review by markkovari — APPROVED** [review]

**Review by AnnaUgrai — APPROVED** [review]

**Pull request merged**